service/controller/clusterapi/v26/key/key.go

```diff
@@ -87,6 +87,11 @@ const (
 	LegacyLabelCluster = "cluster"
 )
 
+const (
+	IngressControllerInsecurePort = 30010
+	IngressControllerSecurePort   = 30011
```

Comment:
The issue desc talks about etcd ports. Should that be ingress ports?

Reply:
Damn right eagle eye.